### PR TITLE
Supplier delete component

### PR DESCRIPTION
**Pull request by Muh-Baharuddin**
@sidratul please review my code. change confirm delete with react-confirm-alert also add error api handle for delete

### CODE REVIEW

File: src/containers/Supplier/Components/DeleteComp.tsx [hunk 1]
```diff
@@ -1,29 +1,47 @@
...
+            .delete(`${process.env.API_ENDPOINT}supplier/` + id)
+            .then((response) => {
+              toast.success(response.data.message, { autoClose: 1500 });
+              setTimeout(() => {
+                window.location.reload();
+              }, 2000);
+            })
+            .catch(() => {
+              toast.error("Maaf sepertinya terjadi kesalahan pada server. Mohon coba kembali dalam beberapa saat.");
+            });
```
Comment (sidratul) [inline src/containers/Supplier/Components/DeleteComp.tsx:30]: create this as function

File: src/containers/Supplier/Components/DeleteComp.tsx [hunk 2]
```diff
@@ -1,29 +1,47 @@
...
-        .delete(`${process.env.API_ENDPOINT}supplier/` + id)
-        .then((response) => {
-          alert(response.data.message)
-          window.location.reload()
-        })
-    }
-  }
+    confirmAlert({
+      title: 'Konfirmasi',
+      message: 'Apakah Anda yakin ingin menghapus data ini?',
```
Comment (sidratul) [inline src/containers/Supplier/Components/DeleteComp.tsx:15]: speciefied the message

File: src/containers/Supplier/Components/DeleteComp.tsx [hunk 3]
```diff
@@ -1,29 +1,47 @@
...
+        {
+          label: 'Ya',
+          onClick: () => {
+            axios
+            .delete(`${process.env.API_ENDPOINT}supplier/` + id)
+            .then((response) => {
+              toast.success(response.data.message, { autoClose: 1500 });
+              setTimeout(() => {
+                window.location.reload();
+              }, 2000);
```
Comment (sidratul) [inline src/containers/Supplier/Components/DeleteComp.tsx:26]: why should reload data? 
just refresh the data.
create context table, contain reload data. when this delete, fetch suppliers with current query

File: src/containers/Supplier/Table/Components/DeleteComp.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
+import axios from "axios"
+import { confirmAlert } from 'react-confirm-alert';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-confirm-alert/src/react-confirm-alert.css';
+import 'react-toastify/dist/ReactToastify.css';
+import { Supplier } from "../TableSupplier";
+
+type Props = {
+  supplier: Supplier;
+  loadData: () => void;
```
Comment (sidratul) [inline src/containers/Supplier/Table/Components/DeleteComp.tsx:10]: refreshSupplier

File: src/containers/Supplier/Components/DeleteComp.tsx (hunk 1 above)
Comment (Muh-Baharuddin) [reply src/containers/Supplier/Components/DeleteComp.tsx]: done, make it as a function

**Review by sidratul — APPROVED** [review]

File: src/containers/Supplier/Components/DeleteComp.tsx (hunk 2 above)
Comment (Muh-Baharuddin) [reply src/containers/Supplier/Components/DeleteComp.tsx]: done

File: src/containers/Supplier/Components/DeleteComp.tsx (hunk 3 above)
Comment (Muh-Baharuddin) [reply src/containers/Supplier/Components/DeleteComp.tsx]: done

File: src/containers/Supplier/Table/Components/DeleteComp.tsx (hunk 4 above)
Comment (Muh-Baharuddin) [reply src/containers/Supplier/Table/Components/DeleteComp.tsx]: done

**Pull request merged**